subprojects/build-init/src/main/java/org/gradle/buildinit/plugins/internal/JvmProjectInitDescriptor.java

```diff
@@ -214,14 +212,20 @@ private void addDependencyConstraints(BuildScriptBuilder buildScriptBuilder) {
         buildScriptBuilder.implementationDependencyConstraint("Define dependency versions as constraints", "org.apache.commons:commons-text:" + commonsTextVersion);
 
         if (getLanguage() == Language.GROOVY) {
-            buildScriptBuilder.implementationDependencyConstraint(null, groovyModuleDependency("groovy-all", libraryVersionProvider.getVersion("groovy")));
+            buildScriptBuilder.implementationDependencyConstraint(null, getGroovyDependency(false));
```

Comment:
How come we're not using `groovyModuleDependency` here anymore?

Reply:
Great point. Looking more closely at this, we can revert all these changes.  These were a workaround for https://issues.apache.org/jira/browse/GROOVY-10543.